### PR TITLE
CSV reporting MODEUR-15 MODEUR-18

### PR DESCRIPTION
**Pull request by adamdickmeiss**
This adds three service:

* `/eusage-reports/stored-reports/use-over-time/csv` 
* `/eusage-reports/stored-reports/reqs-by-date-of-use/csv`
* `/eusage-reports/stored-reports/reqs-by-pub-year/csv`

### CODE REVIEW

File: src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java [hunk 1]
```diff
@@ -1218,7 +1222,101 @@ public void addStartDates(Tuple tuple) {
...
+    writer.print("Reporing period total");
+    JsonArray accessCountPeriods = json.getJsonArray("accessCountPeriods");
+    for (int i = 0; i < accessCountPeriods.size(); i++) {
+      // TODO .. If year  prefix with "Published "
+      // MMMM-DD should be converted to "MON-YYYY"
+      writer.print(accessCountPeriods.getString(i));
+    }
+    writer.println();
+
+    writer.print(
```
Comment (skoczko) [inline src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java:1271]: @adamdickmeiss I think we should remove the row with comments on the field -- they are meant as a helper for the sample report but we probably don't need them in the actual data.

**Review by skoczko — CHANGES_REQUESTED** [review]

File: src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java (hunk 1 above)
Comment (adamdickmeiss) [reply src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java:1271]: ok. will do.

**Comment by adamdickmeiss** [comment]
YYYY-MM ? or Mon-YYYY (mind you that Mon is language specific).

**Comment by skoczko** [comment]
YYYY-MM for months sounds good since it't language independent

**Review by skoczko — APPROVED** [review]

**Pull request merged**